### DISCUSSION TITLE
Bridging the Gap: ASVS and Practical Application

### DISCUSSION THREAD

**Discussion by dn281090pdi** [OWASP/ASVS#2443] [Q&A]
The ASVS checklist contains many technical details that are difficult for non-security specialists to interpret. This creates difficulties both at the design stage, when it is necessary to formulate security requirements, and at the testing stage, when it is necessary to verify their implementation. Linking each ASVS item to related OWASP projects, such as WSTG, CheetSheets, etc, where examples of secure design and verification options are provided, would greatly simplify the use of the standard and make it more accessible to a wider range of professionals.

**Comment by elarlang** [comment]
We are not mapping each requirement to some other project - maybe it is or will be done by CRE project. In ASVS, we link related OWASP project for each chapter in a "References" section.

As it requires fixing case-by-case please point out precise terms that are not understandable.
